lgc/patch/PatchEntryPointMutate.cpp

```diff
@@ -826,7 +826,13 @@ void PatchEntryPointMutate::processCalls(Function &func, SmallVectorImpl<Type *>
         newCalledVal = builder.CreateBitCast(calledVal, calledPtrTy);
       // Create the call.
       CallInst *newCall = builder.CreateCall(calledTy, newCalledVal, args);
-      newCall->setCallingConv(call->getCallingConv());
+      // Change calling convention to amdgpu_gfx if it was spir_func
+      if (call->getCallingConv() == CallingConv::SPIR_FUNC)
```

Comment:
I don't think we should be mentioning SPIR_FUNC in here -- LGC is API-independent. Could you do it by detecting a call that is not to an LGC internal function or an intrinsic?